src/webapi/uiactionscontroller.cpp

```diff
@@ -25,7 +28,8 @@ WebApiResponse UiActionsController::infoAction(WebApiRequest request){
     WebApiResponse response = getResponse();
 
     UiInfoData data = {
-        getMainWindow()->getMapWidget()->zoom()
+        getMainWindow()->getMapWidget()->zoom(),
+        getNavApp()->getWebController()->getWebMapController()->getMapPaintWidget()->zoom()
```

Comment:
This is the line that Irequired for web ui map zoom value retrieval. I had to create `WebController.getWebMapController()`and `WebMapController.getMapPaintWidget()` for it